source/sec_community_communication.ptx

```diff
@@ -148,14 +148,14 @@
         <statement>
           <p>
             The term 
-            <term>merge request</term> or 
-            <term>pull request (PR)</term> describes a request to initiate the process of requesting an integration of specific code changes into the project repository. 
+            <term>merge request (MR)</term> or 
+            <term>pull request (PR)</term> describes a request to initiate the process of integrating specific code changes into the project repository. 
           </p>
         </statement>
       </definition>
                     <p>
 
-          Best practice before making a merge request is to have initiated an earlier discussion using one of the forms of communication favored by the project. After this discussion, a (potential) contributor opens a merge request, which may then undergo several iterations of automatic checks, manual code reviews by one or more other contributor, as well as further discussion.
+          Best practice before making a merge request is to have initiated an earlier discussion using one of the forms of communication favored by the project. After this discussion, a (potential) contributor opens a merge request, which may then undergo several iterations of automatic checks, manual code reviews by one or more other contributors, as well as further discussion.
```

Comment:
Here it might be also worth mentioning that the standard practice is to open an issue after the discussion, and then open a pull request to address said issue. Unless you mean that the discussion itself happens on a newly created issue, in which case I'd be a bit more explicit about it. Something like so:
```suggestion
          Best practice before making a merge request is to have initiated an earlier discussion using one of the forms of communication favored by the project (e.g. by opening an issue on the project's issue tracker). After this discussion, a (potential) contributor opens a merge request, which may then undergo several iterations of automatic checks, manual code reviews by one or more other contributors, as well as further discussion.
```